src/main/scala/IntegrationTests.scala

```diff
@@ -0,0 +1,36 @@
+package tests
+
+import org.apache.spark.SparkContext
+import pipelines.images.cifar.RandomPatchCifar
+import pipelines.images.cifar.RandomPatchCifar.RandomCifarConfig
+import pipelines.images.voc.VOCSIFTFisher
+import pipelines.images.voc.VOCSIFTFisher.SIFTFisherConfig
+import pipelines.text.NewsgroupsPipeline
+import pipelines.text.NewsgroupsPipeline.NewsgroupsConfig
+
+object IntegrationTests {
+  val cifar = RandomPatchCifar.run(_: SparkContext,
+    RandomCifarConfig(
+      trainLocation="./cifar_train.bin",
```

Comment:
Hmm that makes sense. But it might still make sense to use an absolute path for local file locations defined by a program argument or environment variable or something like that? Otherwise I think where we execute the integration tests from could bite us.